scripts/python/tao_build.py

```diff
@@ -578,7 +578,7 @@ def parse_args():
     parser.add_argument(
         "--compiler-gcc",
         required=False,
-        choices=["7.3", "default"],
+        choices=["7.3", "7.5", "default"],
```

Comment:
The default gcc version is 7.5 in devel Docker, but I approved adding an explicit flag here.